client/blocks/reader-full-post/wp-iframe-resize.js

```diff
@@ -0,0 +1,83 @@
+let wrapper;
+
+const receiveEmbedMessage = function ( e ) {
+	const data = e.data;
+
+	if ( ! data ) {
+		return;
+	}
+
+	if ( ! ( data.secret || data.message || data.value ) ) {
+		return;
+	}
+
+	if ( /[^a-zA-Z0-9]/.test( data.secret ) ) {
+		return;
+	}
+
+	const iframes = wrapper.querySelectorAll( 'iframe[data-secret="' + data.secret + '"]' );
+	const blockquotes = wrapper.querySelectorAll( 'blockquote[data-secret="' + data.secret + '"]' );
+
+	// Hide the blockquotes that come together with the iFrame.
+	for ( let i = 0; i < blockquotes.length; i++ ) {
+		blockquotes[ i ].style.display = 'none';
+	}
+
+	for ( let i = 0; i < iframes.length; i++ ) {
+		const source = iframes[ i ];
+
+		if ( e.source !== source.contentWindow ) {
+			continue;
+		}
+
+		source.removeAttribute( 'style' );
+
+		/* Resize the iframe on request. */
+		if ( 'height' === data.message ) {
+			let height;
+			height = parseInt( data.value, 10 );
+			if ( height > 1000 ) {
+				height = 1000;
+			} else if ( ~~height < 200 ) {
```

Comment:
I don't know what the `~~` does here even after looking it up on StackOverflow. It's an alternative to `Math.floor` but here `height` is always an integer because it was returned by `parseInt`.

Reply:
Looking at it more thoroughly doesn't make sense to me either. It seems it was added here https://core.trac.wordpress.org/changeset/35478 as an effort to make sure that `height` is a number.

Thinking it a bit more, I am not sure if we even need to limit the `height` to `200 <= height <= 1000`
How would you feel removing the conditionals and just do `source.height = parseInt( data.value, 10 );`

Reply:
Done here be29e42

Reply:
@cpapazoglou Did you see my comment here? https://github.com/Automattic/wp-calypso/pull/54410#discussion_r669523230